pkg/nfs/controllerserver.go

```diff
@@ -0,0 +1,23 @@
+package nfs
+
+import (
+	"github.com/container-storage-interface/spec/lib/go/csi"
+	"github.com/kubernetes-csi/drivers/pkg/csi-common"
+	"golang.org/x/net/context"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+type ControllerServer struct {
+	*csicommon.DefaultControllerServer
+}
+
+func (cs ControllerServer) ControllerExpandVolume(ctx context.Context, req *csi.ControllerExpandVolumeRequest) (*csi.ControllerExpandVolumeResponse, error) {
+	return nil, status.Error(codes.Unimplemented, "")
+}
+
+func getControllerServer(csiDriver *csicommon.CSIDriver) ControllerServer {
+	return ControllerServer{
+		csicommon.NewDefaultControllerServer(csiDriver),
```

Comment:
I don't think we've updated csicommon for csi spec 1.1 (and we don't intend to. We want to deprecate csicommon).

Reply:
Are we still trying to move csicommon to [kubernetes-csi/csi-lib-common](https://github.com/kubernetes-csi/csi-lib-common)?

Reply:
No we're just going to deprecate it. It doesn't seem to provide much value.

Reply:
okay, also should we stick to csi spec 1.0?

Reply:
1.1 is fine too. The resizing stuff will be unimplemented